content/en/docs/ops/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Upgrade with Helm
+description: How to upgrade Ambient Mesh with Helm.
+weight: 5
+owner: istio/wg-environments-maintainers
+test: yes
+---
+
+Follow this guide to upgrade and configure ambient mesh using
+[Helm](https://helm.sh/docs/).  This guide assumes you have already performed an ambient mesh
+[installation with Helm](/docs/ops/ambient/install/helm-installation/) with a previous minor or patch version of Istio.
+
+{{< boilerplate ambient-alpha-warning >}}
+
+{{< warning >}}
+`Ambient` does not currently support canary upgrades of the dataplane. This means that ambient mesh dataplane upgrades will disrupt application pod traffic. Node cordoning and blue/green node pools are recommended to mitigate blast radius risk during production upgrades. See your Kubernetes provider documentation for details.
+{{< /warning >}}
+
+{{< boilerplate helm-preamble >}}
+
+{{< boilerplate helm-prereqs >}}
+
+## Prerequisites
```

Comment:
Two `Prerequisites` sections.
![image](https://github.com/istio/istio.io/assets/2752495/e6d1bf61-07a5-445c-a87c-49adf68cf50a)

If you want to just append two items to the boilerplate list:

```suggestion
{{< boilerplate helm-prereqs >}}
```

You'll also need to indent the helm command in step 4 of the boilerplate or the numbering will be off/

Reply:
Even with the indent it's not numbering correctly - I'm just going to remove the Helm boilerplates as duplicative from the upgrade guide, since we already say to follow the install guide first and the boilerplates are present there already.